easybuild/easyblocks/o/openfoam.py

```diff
@@ -155,14 +155,21 @@ def configure_step(self):
             self.log.debug("Patching out hardcoded $WM_* env vars in %s", script)
             # disable any third party stuff, we use EB controlled builds
             regex_subs = [(r"^(setenv|export) WM_THIRD_PARTY_USE_.*[ =].*$", r"# \g<0>")]
+
+            # this does not work for OpenFOAM Extend lower than 2.0
+            if 'extend' not in self.name.lower() or LooseVersion(self.version) >= LooseVersion('2.0'):
+                # replacement dictionary of key-values
+                replacement_dict = {"WM_PROJECT_VERSION": self.version}
+                for key, val in replacement_dict.items():
+                    regex_subs += [(r"^(setenv|export) %s=.*$" % key, r"export %s=%s #\g<0>" % (key, val))]
```

Comment:
@mboisson Since `replacement_dict` only has a single entry, this can be collapsed to:

```python
if 'extend' not in self.name.lower() or LooseVersion(self.version) >= LooseVersion('2.0'):
    key = 'WM_PROJECT_VERSION'
    regex_subs.append((r"^(setenv|export) %s=.*$" % key, r"export %s=%s #\g<0>" % (key, self.version)))

Reply:
If this makes the line too long, split it on the `.`:

```python
regex_subs.append((r"^(setenv|export) %s=.*$" % key,
                   r"export %s=%s #\g<0>" % (key, self.version)))